### DISCUSSION TITLE
wide view angle images

### DISCUSSION THREAD

**Discussion by javi-mr** [pyvista/pyvista#3582] [Q&A]
Hi there!

I am trying to configure the camera (pl.camera) as a hemispherical camera (with a 180º viewing angle) but I get a high distortion in the output image. Is it possible to configure the camera with an azimuthal projection to obtain fisheye images?

Thank you!

**Comment by banesullivan** [comment]
I'm wondering if the [`vtkPanoramicProjectionPass`](https://vtk.org/doc/nightly/html/classvtkPanoramicProjectionPass.html#details) rendering pass would get you what you want. Currently, rendering passes are set up as a private API in PyVista though there is likely a way to tap into it and experiment with this rendering pass.

@akaszynski is the most knowledgeable on PyVista's rendering passes at this time, perhaps he can help?

**Reply by MatthewFlamm** [reply]
I think the camera's `view_angle` will be important here too.  I'm not sure how it would interact with a panorama render.
